src/main/java/com/contrastsecurity/sdk/scan/CodeArtifacts.java

```diff
@@ -0,0 +1,58 @@
+package com.contrastsecurity.sdk.scan;
+
+import com.contrastsecurity.exceptions.HttpResponseException;
+import com.contrastsecurity.exceptions.ResourceNotFoundException;
+import com.contrastsecurity.exceptions.ServerResponseException;
+import com.contrastsecurity.exceptions.UnauthorizedException;
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ * Code artifact resource collection.
+ *
+ * <p>This resource API entry point does not use the {@code define()} pattern for creating new code
```

Comment:
I recognized this was really the only resource collection without this pattern and was going to ask.  👍

Reply:
Do you also think it makes sense to introduce an exception to the rule here, or just use `define().file(...).create()` anyway even though it's a bit wordy and annoying?